src/components/Quiz/QuizWidget/useQuizWidget.tsx

```diff
@@ -46,18 +45,9 @@ export const useQuizWidget = ({
     setUserQuizProgress([])
     setShowAnswer(false)
 
-    const currentQuizKey =
-      quizKey ||
-      Object.keys(allQuizzesData).filter((quizUri) =>
-        window?.location.href.includes(quizUri)
-      )[0] ||
```

Comment:
I did a search and it seems that we always pass a `quizKey`, this default is never used. I'm ok with removing it if not used, but just curious about its original purpose, do you know it?

Reply:
I'm honestly not sure... but that was my same reasoning... there was no circumstance where the fallback would be used since a valid key is required.

Reply:
Thinking a little here, it may have simply been a product of not having proper typing, so any `string` could be used which may-or-may-not be available. This shouldn't be an issue now.